Dockerfile

```diff
@@ -5,7 +5,7 @@
 ##    docker build --no-cache --target certs -t vela-email:certs .    ##
 ########################################################################
 
-FROM alpine as certs
+FROM alpine@sha256:51b67269f354137895d43f3b3d810bfacd3945438e94dc5ac55fdac340352f48 as certs
```

Comment:
i think we should add version, as we do here https://github.com/go-vela/server/blob/5e1a7d315e6d36894f7c4b5446e9c477d278ba28/Dockerfile#L3 for example. Note the `3.18.4`, though in this case i think we're dealing with `3.18.5` since the sha256 is different.

Reply:
I added those changes!